website/content/api-docs/system/internal-counters.mdx

```diff
@@ -7,7 +7,7 @@ description: >-
 
 # `/sys/internal/counters`
 
-The `/sys/internal/counters` endpoints are used to return data about the number of Tokens and Entities in Vault. They return information for the entire cluster.
+The `/sys/internal/counters` endpoints are used to return data about the number of Clients and Entities in Vault. They return information for the entire cluster.
```

Comment:
I think Clients and Entities would mean the same thing here. What do you feel @divyaac ?
```suggestion
The `/sys/internal/counters` endpoints are used to return data about the number of Clients in Vault. They return information for the entire cluster.
```

Reply:
Yes agreed.